ndmg/stats/qa_skullstrip.py

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env python
+
+# Copyright 2016 NeuroData (http://neurodata.io)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+# Created by Vikram Chandrashekhar.
+# Edited by Greg Kiar, Eric Bridgeford and Chuankai Luo.
+# Email: Greg Kiar @ gkiar@jhu.edu   cluo16@jhu.edu
+
```

Comment:
clean up this boilerplate intro code to be more similar to all the other files in `staging-tmp` (I removed it since we already have a license at the root)

see our [ndmg-bids](https://github.com/NeuroDataDesign/ndmg/blob/qa_skullstrip_pr/ndmg/scripts/ndmg_bids.py#L3), for instance

Reply:
fixed it